test/cloudtest/test_storaged_shared_fate.py

```diff
@@ -0,0 +1,219 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import subprocess
+import time
+from textwrap import dedent
+
+from materialize.cloudtest.application import MaterializeApplication
+
+CLUSTER_SIZE = 8
+NUM_SOURCES = 4
+
+
+def populate(mz: MaterializeApplication, seed: int) -> None:
+    create_sources = "\n".join(
+        f"""
+            > CREATE SOURCE s{i}
+              IN CLUSTER storaged_shared_fate
+              FROM KAFKA CONNECTION kafka (TOPIC 'testdrive-storaged-shared-fate-${{testdrive.seed}}')
```

Comment:
Going forward we need to use `compute` and `storage` , without the trailing `d`. The reason for the trailing `d` is that before (when the `test_computed_shared_fate.py` was written), there were actual separate `computed` and `storaged` binaries that were being spawned. But currently, there is only a `clusterd` binary for both purposes, so the use of a suffix `d` would be confused unless applied to `computed`.